src/celeritas/ext/detail/GeantPhysicsList.cc

```diff
@@ -307,7 +320,7 @@ void GeantPhysicsList::add_e_processes(G4ParticleDefinition* p)
         }
 
         if (options_.msc == MscModelSelection::wentzel_vi
-            || options_.msc == MscModelSelection::all)
+            || options_.msc == MscModelSelection::urban_wentzel)
```

Comment:
The two `ifs` here cover the cases where we're using either urban, wentzel, or both, but we're missing an `if (options_.msc == MscModelSelection::urban_extended) { ... }`. The `MscEnergyLimit` is overriden at construction but we never use it to assign the process correctly here.

Reply:
I think the fix above takes care of this.